libmpv/qthelper.hpp

```diff
@@ -97,8 +97,8 @@ static inline QVariant node_to_variant(const mpv_node *node)
 }
 
 struct node_builder {
-    node_builder(const QVariant& v) {
-        set(&node_, v);
+    node_builder(const QVariant& v, bool force_list = false) {
```

Comment:
I can; it however, solves an issue where if you use: command(QVariantList{"command"}); internally it gets coerced to QString--this parameter helps force the node_builder to treat it as a list for command.